packages/web/components/EmptyScreen/__tests__/index.js

```diff
@@ -46,8 +55,11 @@ describe('<EmptyScreen />', () => {
 		    class="c1"
 		  >
 		    <img
-		      alt="Ilustração de um rapaz despejando uma caixa aberta em outra caixa"
+		      alt=""
```

Comment:
You haven't removed this attribute 🤔

Reply:
👍🏻

Reply:
I had to remove it in mock because eslint was complaining about it:

```
jest.mock('next/image', () => {
	return {
		__esModule: true,
		// eslint-disable-next-line jsx-a11y/alt-text
		default: (props) => <img {...props} />,
	};
});

```